### PR TITLE
Fix qnap state graphs

### PR DESCRIPTION
**Pull request by TheGreatDoc**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/qnap.yaml [hunk 1]
```diff
@@ -69,10 +69,10 @@ modules:
                     state_name: hdStatus
                     states:
                         - { value: 0, generic: 0, graph: 0, descr: 'Ready' }
-                        - { value: -5, generic: 3, graph: -5, descr: 'No Disk' }
-                        - { value: -6, generic: 2, graph: -6, descr: 'Invalid' }
-                        - { value: -9, generic: 2, graph: -9, descr: 'RW Error' }
-                        - { value: -4, generic: 3, graph: -4, descr: 'Unknown' }
+                        - { value: -5, generic: 3, graph: 0, descr: 'No Disk' }
```
Comment (Jellyfrog) [inline includes/definitions/discovery/qnap.yaml:72]: It would be cool if someone could document what `graph` values actually mean :)

Comment (TheGreatDoc) [reply includes/definitions/discovery/qnap.yaml:72]: Yeah, that way I would not did it wrong in the first place!!! hahaha

I dont have idea on what graph means. I just made the synology move to yaml discovery and found that I did it wrong in the qnap.

Comment (murrant) [reply includes/definitions/discovery/qnap.yaml:72]: graph does nothing :D  AFAIK it was "for future use"

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-67-release-changelog-august-2020/13285/1

**Pull request merged**